### DISCUSSION TITLE
Bell AI & Data Engineering Digital Interview

### DISCUSSION THREAD

**Discussion by iRubeel** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#146] [General]
Just curious to know if anyone got selected for the digital interview at Bell for the position of AI & Data engineering and if anyone could recommend on how to prep for it?

**Comment by KrishPatel13** [comment]
Did you get the email at 10.02 am today?

**Reply by iRubeel** [reply]
nah, not today. I got it yesterday

**Reply by KrishPatel13** [reply]
I thought it would have been automatic. But now it looks like shortlisted people got it. Not sure still.

**Comment by noahserhan** [comment]
> Just curious to know if anyone got selected for the digital interview at Bell for the position of AI & Data engineering and if anyone could recommend on how to prep for it?

It was just a behavioural - nothing special. 2 minutes per question and no redos.

**Comment by ShevinuM** [comment]
> Just curious to know if anyone got selected for the digital interview at Bell for the position of AI & Data engineering and if anyone could recommend on how to prep for it?

I did it some time back. It's an easy behavioral interview. Nothing to stress about.
